.coderabbit.yaml

```
@@ -24,6 +24,27 @@ reviews:
   # Auto-review configuration
   auto_review:
     enabled: true
+  
+  # Custom instructions for AI reviewer - specific to A Lo Cubano Boulder Fest
+  instructions: |
+    As the AI code reviewer for the 'A Lo Cubano Boulder Fest' project, maintain high quality, performance, security, and testability for this Cuban salsa festival website.
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)